### DISCUSSION TITLE
Why are some of the brand icons so weird looking?

### DISCUSSION THREAD

**Discussion by dawidseipold** [tabler/tabler-icons#331] [Q&A] [answered]
Just like in the title. For example, Google or Amazon, Discord, etc. Is there any reason behind this like copyrights or something? It just doesn't look good, especially Google and Discord.

**Comment by codecalm** [comment, marked as answer]
hi @dawidseipold! All icons are in 24x24px grid, and 2px stroke. It's impossible to create some icons with 1:1 brand look. For example, look how original Amazon logo looks in Tabler Icons style:

![Screenshot 2022-10-10 at 16 51 41](https://user-images.githubusercontent.com/1282324/194894457-b3846456-c0e3-4888-b738-5882255c9166.png)
